packages/webos/CHANGELOG.md

```diff
@@ -23,6 +23,11 @@ No significant changes.
 ## [4.7.11] - 2024-05-13
 
 No significant changes.
+## [unreleased]
```

Comment:
Could you please check the CHANGELOG.md? I think it inserted in the wrong place.

Reply:
Well, git cannot merge as humans expect. 😢 Thanks for pointing out.